app/_config/app.yml

```diff
@@ -1,3 +1,3 @@
 Silverstripe\SiteConfig\SiteConfig:
   extensions:
-    - App\extensions\FooterConfig
+    - App\Extensions\ExtraConfig
```

Comment:
Its convention to suffix filenames with `Extension` and have some form of relevance to the config you are extending

I would rename this to `SiteConfigExtension` :) 

Ensure you update the classes below and all other references